### ISSUE TITLE
Make timeout configurable

### ISSUE THREAD

**Issue by leezu** [cpitclaudel/biblio.el#39]
`biblio-url-retrieve` hardcodes a timeout.

https://github.com/cpitclaudel/biblio.el/blob/39c362b9eb36e8c00efb5aa6ed406ef8c6a2e2ac/biblio-core.el#L209-L217

Accessing google-scholar via biblio-gscholar.el from https://github.com/nchodosh/biblio.el/commit/1c532b5b6ce73124aa579cf2c0540c429da00bb1 will often take longer than 1 second and trigger the timeout.

**Comment by cpitclaudel** [comment]
Good point, thanks!  I will look into preparing a patch, unless you're interested in making a PR?

**Comment by leezu** [comment]
I'm fine with you preparing the patch. It may cost you less time given your familiarity with the codebase. Thank you for looking into it :)
